app/public/index.php

```diff
@@ -1,7 +1,5 @@
 <?php
 require_once __DIR__ . '/../vendor/autoload.php';
-use Silvanusp\AtoMap;
```

Comment:
basically this use statement was incomplete, it needs to be the full namespace+classname to be able to find it.